roles/openshift_sdn/files/sdn.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     matchLabels:
       app: sdn
   updateStrategy:
-    type: RollingUpdate
```

Comment:
Restarting the `sdn` pod isn't disruptive, so you could keep this RollingUpdate if you wanted to.

Reply:
It needs to be the `ovs` daemonset.

Reply:
@squeed It seems like the safety net of having a controlled rollout would be nice for both even if not required today?